geode-core/src/main/java/org/apache/geode/management/internal/JmxManagerAdvisor.java

```diff
@@ -102,13 +103,13 @@ protected Profile instantiateProfile(InternalDistributedMember memberId, int ver
    * Overridden to also include our profile. If our profile is included it will always be first.
    */
   @Override
-  protected List/* <Profile> */ fetchProfiles(Filter f) {
+  protected List/* <Profile> */ fetchProfiles(Predicate<Profile> f) {
```

Comment:
Can we fix this raw type here?

Reply:
Tried this, more involved than expected.